### ISSUE TITLE
[Bug] va-alert-expandable needs meaningful SR text to display status

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/vets-design-system-documentation#5050]
## What happened

The icons in the va-alert component do not expose meaningful text equivalents.

Currently, the <va-icon> used as the alert icon (e.g., warning, error, success) is purely decorative from a screen reader’s perspective. The screen reader does announce "Alert" (which not visible as sr-only text) and the visible text in the component, such as:

> Alert: You can fill out a shorter application

Sighted users, however, receive additional meaning through both icon and color treatment. Because the icon and color conveys status (e.g., warning, error, success), the absence of a text equivalent results in a loss of information for assistive technology users.

## What I expected to happen

Screen reader users should hear a message equivalent in meaning to the visual presentation.

For example:

> **Success Alert** You can fill out a shorter application.

Visually this looks like 

> ✅  You can fill out a shorter application.

This provides parity of experience between sighted and non-sighted users.

## Steps to reproduce:

1. [Go to `<va-alert-expandable>` with visible status icon in storybook](https://design.va.gov/storybook/iframe.html?globals=&id=components-va-alert-expandable--success&viewMode=story)
2. Turn on a screen reader (e.g., NVDA or VoiceOver).
3. Move focus to the alert.
4. Listen to what’s announced.

**Actual:**
Only the generic "Alert" status and the visible text is announced.

**Expected:**
The alert’s type is also announced (e.g., “Warning”, “Success”, “Error”) via SR-text.

```
<span class="alert-expandable-title">
   <span class="sr-only">{alert.type} Alert:&nbsp;</span>
      {this.trigger}
   </span>
</span>
```

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other

## Acceptance criteria
- [ ] In va-alert-expandable, screen readers announce the alert type before the message text.
- [ ] SR-only text is visually hidden but accessible via programmatic name.
- [ ] No duplicated announcement (i.e., icon + text doesn’t cause double reading).
- [ ] Implementation verified across browsers and AT combinations (NVDA + Firefox, JAWS + Chrome, VoiceOver + Safari).
- [ ] Warning alerts announce "Warning Alert"
- [ ] Error alerts announce "Error Alert"
- [ ] Success alerts announce "Success Alert"
- [ ] Informational alerts announce "Information Alert"

## Related issue
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/5046
